scripts/copy_from_upstream/patches/pqclean-kyber-armneon-yml.patch

```diff
@@ -0,0 +1,60 @@
+From 8127443600b88b8b98836b5f2f74a7995e9d5e7b Mon Sep 17 00:00:00 2001
+From: Martyrshot <Martyrshot@gmail.com>
+Date: Sun, 31 Oct 2021 04:37:35 +0000
+Subject: [PATCH] Patch for pqclean's meta yml
+
+---
+ crypto_kem/kyber1024/META.yml | 4 +---
+ crypto_kem/kyber512/META.yml  | 4 +---
+ crypto_kem/kyber768/META.yml  | 5 +----
+ 3 files changed, 3 insertions(+), 10 deletions(-)
+
+diff --git a/crypto_kem/kyber1024/META.yml b/crypto_kem/kyber1024/META.yml
+index 28e2014..8ba4d4e 100644
+--- a/crypto_kem/kyber1024/META.yml
++++ b/crypto_kem/kyber1024/META.yml
+@@ -37,9 +37,7 @@ implementations:
+     - name: aarch64
+       version: https://github.com/neon-ntt/neon-ntt/tree/b011eeff3515fb168aa4dbaa671d760009d98dbb
+       supported_platforms:
+-        - architecture: arm_8
++        - architecture: ARM64_V8
+           operating_systems:
+               - Linux
+               - Darwin
+-          required_flags:
+-              - asimd
+diff --git a/crypto_kem/kyber512/META.yml b/crypto_kem/kyber512/META.yml
+index 3c4d1ee..d1ac307 100644
+--- a/crypto_kem/kyber512/META.yml
++++ b/crypto_kem/kyber512/META.yml
+@@ -37,9 +37,7 @@ implementations:
+     - name: aarch64
+       version: https://github.com/neon-ntt/neon-ntt/tree/b011eeff3515fb168aa4dbaa671d760009d98dbb
+       supported_platforms:
+-        - architecture: arm_8
++        - architecture: ARM64_V8
```

Comment:
What is the reason for changing an architecture's name? Also: Has this been PR'd to upstream (PQClean) so that eventually OQS won't need these patches any more?

Reply:
> Do I understand this PR right in that it enables architecture-specific pulls from different upstream repositories (and doing this for ARM64 Kyber)? 
Yes, exactly.
>It then seems documentation generation needs to be improved as all KEM docs (.yml and .md) are touched _except_ those for Kyber -- which seems weird: Can you please check/amend that suitably?
Will do.
>Only as a question: As this seems to introduce a somewhat different code base for Kyber, would it be worth while/sensible/possible to add constant time tests for this code/platform -- or is this over the top (maybe to be added as a new issue: "constant time checks for aarch64")?
I think as our codebase keeps adding optimized aarch64 implementations this would be worthwhile, but perhaps we should create another issue to discuss this. I'll make a new issue shortly.
>Finally, for curiosity: Is this code markedly faster than the ARM code presently generated? Worthwhile updating [profiling](https://github.com/open-quantum-safe/profiling) with this?
I hope so, but I haven't performed any benchmarking yet. This was a case of @dstebila asking me to pull in pqclean's kyber code and blindly doing it. :-) I've only skimmed the associated [paper](https://eprint.iacr.org/2021/986.pdf), but it looks like there is a decent speed improvement over past kyber implementations.

Reply:
> What is the reason for changing an architecture's name? Also: Has this been PR'd to upstream (PQClean) so that eventually OQS won't need these patches any more?

I patched this so that it matches out naming convention of ARM64_V8. I did this so that the fragment files would match our style. I don't think upstream should be changed to match our naming convention, unless they like our naming better.